api/v1alpha1/performanceprofile_conversion.go

```diff
@@ -0,0 +1,120 @@
+package v1alpha1
+
+import (
+	"github.com/openshift-kni/performance-addon-operators/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+// ConvertTo converts this PerformanceProfile to the Hub version (v1).
+func (curr *PerformanceProfile) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v1.PerformanceProfile)
+
+	// ObjectMeta
+	dst.ObjectMeta = curr.ObjectMeta
+
+	// Spec
+	dst.Spec.CPU = new(v1.CPU)
+	hugePagesSize := v1.HugePageSize(*curr.Spec.HugePages.DefaultHugePagesSize)
```

Comment:
probably you should to check here `if curr.Spec.HugePages != nil && curr.Spec.HugePages.DefaultHugePagesSize != nil`

Reply:
will do, thanks

Reply:
done